src/scripts/background.js

```diff
@@ -85,7 +85,6 @@ window.TogglButton = {
     <div class="TB__Dialog__field" tabindex="0">
       <div>
         <div id="toggl-button-project-placeholder" class="TB__FormFieldTrigger__trigger" disabled><span class="tb-project-bullet"><div>No project</div></span><span class="TB__Popdown__caret"></span></div>
-        <div class="TB__Popdown__overlay"></div>
```

Comment:
Removed the overlay because it wasn't really necessary and was breaking the capability of clicking on other elements on the page.